### PR TITLE
Added Footer Data

### PR DESCRIPTION
**Pull request by sumitd94**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -22,6 +22,14 @@ const Index = () => {
         />
       </Head>
       <HomePage />
+      <footer>
+        <p>
+          The contents of this website are deployed from this 
+          <a href="{https://github.com/Real-Dev-Squad/website-members}" target="_blank">
```
Comment (ankushdharkar) [inline pages/index.js:28]: No css used?

Comment (sumitd94) [reply pages/index.js:28]: Not yet, I first wanted to make sure this changes work and accordingly next step will be to add css.

Comment (ankushdharkar) [inline pages/index.js:28]: Do you need `{ }` for html attribute?

Comment (sumitd94) [reply pages/index.js:28]: To render the the href link in react we use { } . Will the link work without {} as well ? Let me try and check in my local

Comment (ankushdharkar) [reply pages/index.js:28]: Not good from a PR submission standpoint. Finish adding the CSS please.

Comment (ankushdharkar) [reply pages/index.js:28]: Without the { }, no dynamic content will render. But, our string is static anyway :)

Comment (sumitd94) [reply pages/index.js:28]: Yep ok

Comment (sumitd94) [reply pages/index.js:28]: Ok, I will add the css and re submit the PR

**Comment by ankushdharkar** [comment]
Solves #9

**Comment by ankushdharkar** [comment]
Closed in favour of #9

**Pull request closed without merging**